### PR TITLE
added doc examples for distance_measures

### PR DESCRIPTION
**Pull request by Qudirah**
fixes for #6575

### CODE REVIEW

File: networkx/algorithms/approximation/distance_measures.py [hunk 1]
```diff
@@ -39,6 +39,18 @@ def diameter(G, seed=None):
     d : integer
        Lower Bound on the Diameter of G
 
+    Examples
+    --------
+    >>> from networkx.algorithms.approximation.distance_measures import diameter
```
Comment (rossbar) [inline networkx/algorithms/approximation/distance_measures.py:44]: We want users to call `nx.diameter`, not import the function from this (very deep) package!

**Review by rossbar — CHANGES_REQUESTED** [review]

**Comment by MridulS** [comment]
@Qudirah just wanted to bump this up :)

If you have the time could you please fix up the requested changes in the PR, so I can go through and review them? Thanks for your contributions!!

**Comment by Qudirah** [comment]
@MridulS i just got caught up with something else. Will attend to everything real soon. Sorry.

File: networkx/algorithms/approximation/distance_measures.py [hunk 2]
```diff
@@ -39,6 +39,17 @@ def diameter(G, seed=None):
...
        Lower Bound on the Diameter of G
 
+    Examples
+    --------
+    >>> G = nx.path_graph(10) #undirected graph
+    >>> nx.diameter(G)
+    9
+
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(2,3),(3,5),(5,2)]) #directed graph
```
Comment (dschult) [inline networkx/algorithms/approximation/distance_measures.py:49]: The cycle_graph function can be used for this case:
```suggestion
    >>> G = nx.cycle_graph([2, 3, 5], create_using=nx.DiGraph)  # directed graph
```

**Pull request closed without merging**